### PR TITLE
Fixes CONTRIBUTING.md file issues

### PR DESCRIPTION
**Pull request by kathyoleary**
:green_book:  This one is ready to go.  Need some thumbs please!

### CODE REVIEW

**Comment by rylnd** [comment]
It looks like the file is still lowercased. See [this article](http://stackoverflow.com/questions/17683458/how-do-i-commit-case-sensitive-only-filename-changes-in-git) for instructions on how to change the case of a file in git.

File: README.md [hunk 1]
```diff
@@ -30,7 +30,7 @@ GitHub's [Hello World]
 GitHub's [Open Source Guide]
 GitHub's [Video Training]
 
-[CONTRIBUTING]: https://github.com/LMHT/LMHT.github.io/blob/master/contributing.md  
+[CONTRIBUTING]: https://github.com/LMHT/LMHT.github.io/blob/master/CONTRIBUTING.md
```
Comment (rylnd) [inline README.md]: I would not hard-code the repository that this link points to. If you instead do

``` markdown
[CONTRIBUTING]: CONTRIBUTING.md
```

That should link to the file named 'CONTRIBUTING.md' in whichever repo you're currently it in, regardless of name, user, company, etc.

**Comment by kathyoleary** [comment]
corrected the capitalization and did some changes to the python server paragraph to show it as code and a lot less verbose

**Comment by rylnd** [comment]
:+1:

**Comment by ericanaughton** [comment]
@kathyoleary It seems like the links on the top are not necessary. The document is not incredibly long, and the titles of each section are large enough to scroll through and find what you're looking for. Also, the links within the 'Ways to Contribute' section repeat half of the links just mentioned. I say, keep the links in the Contribute section, but maybe remove those links on the top.

**Comment by ericanaughton** [comment]
@kathyoleary the sections 'submit an issue', 'request a feature' and 'edit documentation' are all very similar. It seems repetitive to explain that the contributor should submit a new issue in three different places, right in a row. What if these three topics are addressed together in one paragraph? Cutting down on words may be helpful to developers who just want to get started on the project.

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -49,6 +50,15 @@ As a contributor, the expectation is that you can use Git and GitHub comfortably
 
 3.  It is common practice to name your branch according to what it is fixing or updating. Before submitting these changes, verify they work. You can include screenshots of the before and after, if the changes are HTML/CSS based, and drop those images into your pull request for further verification. Please try to maintain the look of the site as we currently have it.
 
+#### Verifying Your Changes
+
+  Verify your changes are correct before submitting a pull request by rendering your html doc.  Because the project uses Ajax, you must have a local web server for this to work properly.  If you have OS X:
```
Comment (ericanaughton) [inline CONTRIBUTING.md]: @kathyoleary html should be capitalized (line 55)

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -49,6 +50,15 @@ As a contributor, the expectation is that you can use Git and GitHub comfortably
...
+
+    $  cd myrepo
+    $  python -m SimpleHTTPServer
+    *  Running on [http://0.0.0.0:8000](http://0.0.0.0:8000/)
+
+
 4.  Once you have submitted a pull request, we will review it. Post any changes or suggestions for review or opinion and handle accordingly. Not every pull request will be added to our database. Follow the request to keep up to date on its status.
```
Comment (ericanaughton) [inline CONTRIBUTING.md]: I don't think this paragraph needs to be numbered.

Comment (ericanaughton) [inline CONTRIBUTING.md]: The wording, "...and handle accordingly" seems out of place.

Comment (ericanaughton) [inline CONTRIBUTING.md]: line 69: You could replace the words "..at which point.." with "then"  to cut down on words, for quicker reading.

Comment (ericanaughton) [inline CONTRIBUTING.md]: line 71: I think we've stayed on top of issues pretty well, so I don't think it's necessary to state that "sometimes an issue will stay open..."

**Comment by jremsik-gpsw** [comment]
:+1:

**Comment by kathyoleary** [comment]
You edited out the fourth paragraph and just put the python server in its
place, but, and it may sound stupid, it doesn't actually tell them to
submit their pull request..

**Comment by ericanaughton** [comment]
@kathyoleary my thought is that contributors on Github will already have experience with how to submit a PR. if they're not sure how to do that, then they can check out the links we've provided for further learning :)

**Comment by kathyoleary** [comment]
Yes, but we already put steps 1-3 out there telling them how to make one, just figured we'd have the last part to submit.  Also, noticed that the http: link isn't showing as a clickable link and it being in the parentheses after it in [] should make it such.  Did putting it in a show as coding block stop it from working as such?

**Pull request merged**